addons/sourcemod/translations/vip_core.phrases.txt

```diff
@@ -2,23 +2,23 @@
 {
 	"VIP_CHAT_PREFIX"
 	{
-		"en"		"[VIP]"
-		"fi"		"[VIP]"
-		"de"		"[VIP]"
-		"pt"		"[VIP]"
-		"pt_p"		"[VIP]"
-		"ru"		"[VIP]"
-		"ua"		"[VIP]"
+		"en"		"{DEFAULT}[ {RED}VIP {DEFAULT}]"
```

Comment:
let's use {GREEN} instead of {RED} to support all games, and every user can change what he wants

Reply:
Done